src/hyperlane-adapter/interfaces/ExecutorAware.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Why is this file here?

Reply:
I am using a modified executorAware contract, which allows me to add multiple trusted executors [https://github.com/degencodebeast/ERC5164/blob/main/src/hyperlane-adapter/interfaces/ExecutorAware.sol](url)

Reply:
I am using a modified executorAware contract, which allows me to add and remove multiple trusted executors [https://github.com/degencodebeast/ERC5164/blob/main/src/hyperlane-adapter/interfaces/ExecutorAware.sol](url)

Reply:
> Why is this file here?

I am using a modified executorAware contract, which allows me to add and remove multiple trusted executors [https://github.com/degencodebeast/ERC5164/blob/main/src/hyperlane-adapter/interfaces/ExecutorAware.sol](https://github.com/pooltogether/ERC5164/pull/17/files/url)

Reply:
I am using a modified [ExecutorAware](https://github.com/degencodebeast/ERC5164/blob/main/src/hyperlane-adapter/interfaces/ExecutorAware.sol) contract, which allows me to add and remove multiple trusted executors

Reply:
Is that necessary? It seems useful to be compliant with the default implementation? (its why we are doing this PR in the first place no?)

Reply:
I have removed it